theme/templates/generic/includes/rating.html

```diff
@@ -42,10 +39,6 @@
     {% elif rating_sum <= 0 %}
         +1 this.
     {% endif %}
-
-    {% if not request.user.is_authenticated %}
-        &nbsp;(<a href="{% url 'login' %}">You need to be logged in to rate this.</a>)
-    {% endif %}
```

Comment:
Hmm, removing this means anonymous users who haven't logged in can rate a resource as well? I don't think that is the desired behavior. We do only allow logged in users to rate a resource.

Reply:
In line 167, the Mezzanine initial_validation function checks if a user is authenticated. If not, then it redirects the user to the login page. So, lines 46-48 follows a redundant behavior.

Reply:
Oh, that is good to know. Thanks